src/ontology/mondo-edit.obo

```diff
@@ -351050,7 +351054,7 @@ synonym: "MEB disease with bilateral multicystic leucodystrophy" EXACT [Orphanet
 xref: ICD10CM:G71.2 {source="Orphanet:370997/attributed", source="Orphanet:370997/ntbt", source="Orphanet:370997"}
 xref: Orphanet:370997 {source="MONDO:equivalentTo"}
 is_a: MONDO:0018276 ! muscular dystrophy-dystroglycanopathy
-is_a: MONDO:0018869 ! cobblestone lissencephaly
+is_a: MONDO:0018869 {source="Orphanet:370997"} ! cobblestone lissencephaly
```

Comment:
@nicolevasilevsky can you help me understand why the `is_a` has a source, but in other cases when this class is used in an `excluded_subClassOf` relationship the source is not present?

Reply:
I don't know where all these unsupported annotations came from, we have a ticket for this: https://github.com/monarch-initiative/mondo/issues/5857

We initially thought we could just remove them all but as we started reviewing them, we realized some were legitimate.